examples/esm/chat/index.html

```diff
@@ -16,6 +16,8 @@
     </head>
 
     <body>
```

Comment:
Remove these changes from PR

Reply:
@VenkateswararaoVe-Kore This PR Is not ready for review. will update you when it is ready. Please don't review until informed.